manuscript.md

```diff
@@ -52,6 +52,8 @@
 
 Networks can vary either in their structural properties (e.g. changes in connectance or degree distribution) or in their composition (e.g. changes in nodes and links identity). Interestingly, variation in the structural properties of ecological networks tend to respond mostly to changes in the size of the network. First, the number of links in ecological networks scales with the number of species [@MacDonald2020RevLin; @Brose2004UniSpa]. Second, connectance and size are driving the rest of the structural properties of ecological networks [@Poisot2014WheEco; @Dunne2002FooStr; @Riede2010ScaFoo]. So, network properties do vary in space, but this variation is mainly explained by the spatial variation in species richness.
 
+On a lower level, species turnover in space obviously results in changes in the composition of ecological networks. But, this is not the only reason networks composition varies [@Poisot2015BeySpe]. Intraspecific variations can result in interaction turnovers without changes in species composition [ADD REF]. Similarly, changes in species abundances can lead to variation in interactions [ADD REF].
```

Comment:
> Similarly, changes in species abundances can lead to variation in interactions [ADD REF].

I'm thinking [Vasquez et al 2007 in Oikos](http://www.dx.doi.org/10.1111/j.0030-1299.2007.15828.x)